core/src/main/java/com/linecorp/armeria/common/HttpResponseWriter.java

```diff
@@ -199,4 +221,12 @@ default void respond(AggregatedHttpMessage res) {
 
         close();
     }
+
+    /**
+     * Write the given {@link HttpObject} and close the stream.
+     */
+    default void respond(HttpObject obj) {
```

Comment:
- `respond` -> `close` ?
- Could we move this method to `StreamWriter`? i.e. `default void close(T obj)` If we are worried about the confusion with ` close(Throwable)` then we can copy this method to `HttpRequestWriter` instead.